src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -181,9 +181,12 @@ def register_submission(
 
         betrokkene_identificatie = rol_data.get("betrokkeneIdentificatie", {})
         kvk = betrokkene_identificatie.pop("_kvk", None)
+        vestigingsnummer = betrokkene_identificatie.get("vestigingsNummer")
 
-        if kvk:
+        if kvk and vestigingsnummer:
             rol_data["betrokkeneType"] = "vestiging"
+        elif kvk:
+            rol_data["betrokkeneType"] = "niet_natuurlijk_persoon"
```

Comment:
I'm missing the `rol_mapping` adjustments (used in line 179, and define on line 90):

This is the available items in the Zaken API but I think we can only match a few.

```json
{ 
"innNnpId": "string",
"annIdentificatie": "string",
"statutaireNaam": "string",
"innRechtsvorm": "besloten_vennootschap",
"bezoekadres": "string",
"subVerblijfBuitenland": {
"lndLandcode": "stri",
"lndLandnaam": "string",
"subAdresBuitenland_1": "string",
"subAdresBuitenland_2": "string",
"subAdresBuitenland_3": "string"
}
```